workflow/Utils/utils.go

```diff
@@ -138,22 +128,64 @@ func CreateHTTPClient() *http.Client {
 
 func CreateUrl(BaseUrl string, Port string, APIUrl string) (string, error) {
 
-	if !strings.HasPrefix(BaseUrl, "http://") && !strings.HasPrefix(BaseUrl, "https://") {
-		return "", errors.New("BaseUrl must start with http:// or https://")
-	}
+	finalURL := fmt.Sprintf("%s:%s%s", BaseUrl, Port, APIUrl)
+	return finalURL, nil
+}
+
+func EmailGenerator(orderID uint, orderStatus string) ([]byte, error) {
```

Comment:
method name should be a verb

Reply:
i have changed the function name to GenerateEmail.